### PR TITLE
Task#231

### PR DESCRIPTION
**Pull request by MalteMagnussen**
## Taiga
https://tree.taiga.io/project/maltemagnussen-fog/us/197?milestone=231798

### Part one
image

### Part two
image

## Site when logged in as employee:
image

### CODE REVIEW

File: FogCarport/src/main/java/data/databaseAccessObjects/mappers/UserMapper.java [hunk 1]
```diff
@@ -160,16 +132,25 @@ public EmployeeModel getEmployee(int id) throws UserException
...
+                + "(`customer_name`,\n"
+                + "`phone`,\n"
+                + "`email`,\n"
+                + "`password`,\n"
+                + "`registered`)\n"
+                + "VALUES\n"
+                + "(?,\n"
+                + "?,\n"
+                + "?,\n"
+                + "?,\n"
```
Comment (HrBjarup) [inline FogCarport/src/main/java/data/databaseAccessObjects/mappers/UserMapper.java]: Won't the "\n" that you've added cause problems in the DB?

**Comment by HrBjarup** [comment]
Can you add some screenshots?

**Comment by MalteMagnussen** [comment]
> Can you add some screenshots?

Yes - Coming up.

File: FogCarport/src/main/java/data/databaseAccessObjects/mappers/UserMapper.java (hunk 1 above)
Comment (MalteMagnussen) [reply FogCarport/src/main/java/data/databaseAccessObjects/mappers/UserMapper.java]: https://github.com/HrBjarup/Fog-Carport/pull/140/commits/49128559f2510c08596f36e5c3ff81500c56a1a1

Comment (HrBjarup) [reply FogCarport/src/main/java/data/databaseAccessObjects/mappers/UserMapper.java]: Yep

**Pull request merged**